stellarphot/settings/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .models import *
+from .views import *
```

Comment:
Just so you know, ruff really hates * on imports (and PEP8 isn't terribly pleased with it either).  I am wondering if despite the pain we should be explicit with the function imports.  I know `__all__` is controlling this to some extent, but I wonder if linters are happier in part because they know where to look for class/function definitions in these cases.

Reply:
OK; I had thought this was the one case it was OK to get the namespace the way you want it.